### PR TITLE
feat(51): Fix optional fields for DTOs

### PR DESCRIPTION
**Pull request by hromogen**
- Fix optional fields for DTOs

### CODE REVIEW

**Review by dartwlad — APPROVED** [review]

File: packages/server/src/models/social-profile.dto.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
-import { IsString } from 'class-validator';
+import { IsString, IsOptional } from 'class-validator';
 
 export default class SocialProfileDto {
   @IsString()
   public pictureUrl: string;
 
+  @IsOptional()
   @IsString()
   public facebookId?: string;
```
Comment (dubbha) [inline packages/server/src/models/social-profile.dto.ts:9]: Just curious, what does the question mark affect in this case?

Comment (hromogen) [reply packages/server/src/models/social-profile.dto.ts:9]: Type checking in code. It is Typescript after all)

File: packages/server/src/models/user.entity.ts [hunk 2]
```diff
@@ -27,7 +26,7 @@ export default class User {
   public password: string;
 
   @Column()
-  public status: UserStatuses;
+  public status: string;
```
Comment (dubbha) [inline packages/server/src/models/user.entity.ts]: Why did you decide to abandon using enums? I'm using them heavily after your hint here :)

Comment (dubbha) [reply packages/server/src/models/user.entity.ts]: E.g. as single values and arrays:
https://github.com/dubbha/croton/pull/81/files#diff-d584a668c576ebf201af646c073a277bR8-R14

Comment (hromogen) [reply packages/server/src/models/user.entity.ts]: Well, for some reason it is verified as an Object by `typeorm` `@Column` decorator. The error you have sent in Slack has been caused by that, it seems. All over the code we do use the `enum` though

Comment (dubbha) [reply packages/server/src/models/user.entity.ts]: `@Column('simple-enum')` should work.
But you leave it if you don't want to try.

**Review by dubbha — APPROVED** [review]

**Pull request merged**